python/ray/serve/controller.py

```diff
@@ -119,9 +121,30 @@ async def run_control_loop(self) -> None:
                     self.backend_state.update()
                 except Exception as e:
                     logger.error(f"Exception updating backend state: {e}")
-
+            self._put_serve_snapshot()
             await asyncio.sleep(CONTROL_LOOP_PERIOD_S)
 
+    def _put_serve_snapshot(self) -> None:
+        val = []
+        for name, (backend_info,
+                   route_prefix) in self.list_deployments().items():
+            deployment_info = {}
+            deployment_info["name"] = name
+            deployment_info["namespace"] = "TODO"
+            deployment_info["ray_job_id"] = "TODO"
```

Comment:
After discussing with Simon, with a naive implementation these fields will all be the same across all deployments; they'll just be the job_id and namespace of the Controller.  Is this okay?  Or do we want to somehow allow different job_id's and namespaces per Deployment (for example, if `.deploy()` is called from a different driver script, does one expect the resulting deployment to inherit the job id from that driver script?)